pkg/credentials/provider_test.go

```diff
@@ -7,13 +7,18 @@ import (
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/aws/aws-sdk-go/service/sts/stsiface"
+	"github.com/google/uuid"
 	"github.com/square/spiffe-aws-assume-role/internal/mocks"
 	"github.com/square/spiffe-aws-assume-role/internal/test"
 	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 )
 
+const defaultEndpoint string = ""
```

Comment:
Not a huge fan of Go's constant naming convention but it is what it is.

Reply:
you could also do:
```
const (
  defaultEndpoint = ""
)
```